src/iqstat.cpp

```diff
@@ -208,12 +208,15 @@ Print the main usage/help information.
 void usage() {
     char *msgs[] = {
         "Usage: iqstat [-luvVh] [-u user] [ruleId]",
```

Comment:
Consider making this better. Take a look at `istream` and `itouch`'s help text.

This is not required. Feel free to ignore this comment.

Reply:
I'll give it a go.  oh yes, also, I was conflicted w.r.t  the verbose options.  Due to the way they're processed, `-v` is accepted but does nothing, and `-V` has effects not evident within the client's source code, effects which I feel unprepared to describe, other than "makes the client very verbose."

Reply:
made the arguments stand out a little better via all capitals and aligned everything better, like the aforementioned sample icommand help texts.  Here's what is produced now:
```
$ ./iqstat -h
Usage:     iqstat [-luvVh] [-u USER] [RULE_ID]
 
Show information about pending iRODS rule executions (i.e. delayed rules).
By default, only the invoking user's rules are shown.
 
Options:
 -a        Display requests of all users.
 -l        Force use of the long format.
 -u USER   Display requests for the specified user (may not be used together with -a).
 RULE_ID   Limits the query to the specified integer rule ID (and implies long format).
 
See also iqdel and iqmod

iRODS Version 4.2.11                iqstat

```

Reply:
I like it. Mark this comment as resolved when you're ready.

Reply:
coolness